app/controllers/repositories_controller.rb

```diff
@@ -100,14 +103,21 @@ def branches
   end
 
   def notify_push
```

Comment:
Maybe a `Webhooks` controller? Would it be useful to have actions specifically for each service, like `gitlab_push_nofity`, so in the future we could also have `github_push_notify`?